### ISSUE TITLE
Style "first row" differently (and probably make it stick with the header)

### ISSUE THREAD

**Issue by kktsvetkov** [kktsvetkov/heiho#4]
This script is previewing only CSVs, not real spreadsheets. In almost all of the CSVs the first row is the header row with the titles of the columns. It makes sense to have it styled differently so that is more obvious that this is the row with the column titles. No need for anything flashy, just making them bold might be enough.

Considering #3 perhaps this first row must also stick to the top of the preview. 

All of this is best to be configurable through the options so that there is some options flag that instructs the preview whether to treat the first row differently or not. Some research into how this can be auto-detected will probably help with the default behaviour of this setting when nothing explicitly is specified.

**Comment by kktsvetkov** [comment]
> Here's a few heuristics that would tend to indicate the first line isn't a header:
> 
> * The first row has columns that are not strings or are empty
> * The first row's columns are not all unique
> * The first row appears to contain dates or other common data formats (eg, xx-xx-xx)

https://stackoverflow.com/a/2674078

**Comment by kktsvetkov** [comment]
Python's csv.sniffer.has_header 
```
        # Creates a dictionary of types of data in each column. If any
        # column is of a single type (say, integers), *except* for the first
        # row, then the first row is presumed to be labels. If the type
        # can't be determined, it is assumed to be a string in which case
        # the length of the string is the determining factor: if all of the
        # rows except for the first are the same length, it's a header.
        # Finally, a 'vote' is taken at the end for each column, adding or
        # subtracting from the likelihood of the first row being a header.
```
https://hg.python.org/cpython/file/tip/Lib/csv.py#l384

**Comment by kktsvetkov** [comment]
Sticky first row should be addressed when working on #3
